pkgs/applications/office/ticktick/default.nix

```diff
@@ -15,11 +15,11 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "ticktick";
-  version = "2.0.0";
+  version = "2.0.10";
 
   src = fetchurl {
     url = "https://d2atcrkye2ik4e.cloudfront.net/download/linux/linux_deb_x64/${finalAttrs.pname}-${finalAttrs.version}-amd64.deb";
-    hash = "sha256-LOllYdte+Y+pbjXI2zOQrwptmUo4Ck6OyYoEX6suY08=";
+    sha256 = "0h87nylmyy8786x8dw3731hml8sml9nk49qwzpx7bq399pnjfa62";
```

Comment:
Please keep the existing hash format. The easiest way to do this is set `hash = "";`, then rebuild to get the actual hash.